doc/operator-capabilities.md

```diff
@@ -422,7 +422,7 @@ spec:
 ```
 
 * **NOTE 1**: `httpMethod`, `pattern`, `increment` and `metricMethodRef` fields are required.
-* **NOTE 2**: `metricMethodRef` holds a reference to existing metric or method map key name `system_name`. In the example, `hits`.
+* **NOTE 2**: `metricMethodRef` holds a reference to the existing metric or method map key name `system_name`. In the example, `hits`.
```

Comment:
I think this change should be 'to an existing metric...' instead of 'to the existing metric...'